django/db/migrations/operations/models.py

```diff
@@ -147,6 +150,8 @@ def reduce(self, operation, in_between, app_label=None):
                     managers=self.managers,
                 ),
             ]
+        elif isinstance(operation, CreateModel):
+            return True
```

Comment:
Just because Django doesn't automatically put relational fields inside `CreateModel` operations doesn't mean they are not supported. I can still have a manually created migration with `CreateModel('app', 'A')` and `CreateModel('app', 'B', fields=[('aaa', models.ForeignKey('app.A'))])` that would not be valid to optimize across for `app.A`.